jgrapht-ext/src/main/java/org/jgrapht/ext/DOTExporter.java

```diff
@@ -266,6 +285,20 @@ private String getVertexID(V v)
                 + "' is not valid with respect to the .dot language");
     }
 
+    /**
+     * Test if the ID candidate is a valid ID.
+     *
+     * @param idCandidate the ID candidate.
+     *
+     * @return <code>true</code> if it is valid; <code>false</code> otherwise.
+     */
+    public static boolean isValidID(final String idCandidate) {
```

Comment:
Sorry, I use to include final in all the variables and I wasn't aware that it is not the case here. Fixed.